git-archive-all.sh

```diff
@@ -246,8 +248,9 @@ fi
 if [ $VERBOSE -eq 1 ]; then
     echo -n "archiving submodules..."
 fi
+git submodule >>$TMPLIST
```

Comment:
Ok sure, np. I had the same idea originally but seeing this same pattern (e.g. `>> $TOARCHIVE` a few lines above) scattered all over the file, decided against it :)